invenio_access/models.py

```diff
@@ -167,6 +167,21 @@ def __repr__(self):
         """Repr."""
         return "{0.name} - {0.description}".format(self)
 
+    @staticmethod
```

Comment:
please use the model directly without your custom methods

cons: it is not DRY